omniscripts/scripts/benchmark_run.py

```diff
@@ -1,6 +1,7 @@
 import os
 import time
 
+from omniscripts.timer import tm
```

Comment:
just to be aligned with other places where we import `tm`
```suggestion
from omniscripts import tm
```

Reply:
Surprisingly, this will lead to circular import (omniscripts.__init__->omniscripts.scripts.benchmark_run->omniscripts.__init__.->omniscripts.timer)